certtostore_windows.go

```diff
@@ -1059,7 +1075,7 @@ func (w *WinCertStore) generateECDSA(algID string) (crypto.Signer, error) {
 		uintptr(unsafe.Pointer(wide(algID))),
 		uintptr(unsafe.Pointer(wide(w.container))),
 		0,
-		nCryptMachineKey|nCryptOverwriteKey)
+		w.keyAccessFlags|nCryptOverwriteKey)
```

Comment:
I suspect we're going to have a type issue here with keyAccessFlags being uintptr but nCryptOverwriteKey being int (?).

Reply:
It's fine, because nCryptOverwriteKey is a numeric constant not an int.
Double-checked that by running code on windows laptop :)